src/committee/orchestrator.rs

```diff
@@ -578,13 +578,11 @@ pub async fn run_server(
     let mss_thread_copy = member_status_state.clone();
     tokio::spawn(async move { MemberStatusState::keepalive_thread(mss_thread_copy).await });
 
-    let ctx = (
-        Orchestrator {
-            pubkey_package,
-            committee_cfg,
-        },
-        member_status_state.clone(),
-    );
+    let ctx = Orchestrator {
+        pubkey_package,
+        committee_cfg,
+        member_status: member_status_state,
+    };
```

Comment:
perhaps it would be cleaner to do a `let ctx = Orchestrator::new(pubkey_package, committee_cfg)` and take care of creating the `member_status_state` there

Reply:
Yup